### PR TITLE
PMM-8715 replica setup options for pmm-framework

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: pmm-tests/pmm-framework.sh [hunk 1]
```diff
@@ -106,12 +106,13 @@ usage () {
   echo " --setup-mongodb-ssl            Use this option to setup official mongodb 4.4 server with ssl"
   echo " --setup-postgres-ssl           Use this option to setup official Postgresql 13 with SSL"
   echo " --setup-remote-db              Use this option when running AMI/OVF instances and setting up remote db's on client node"
+  echo " --mongo_replica_for_backup     Use this option to setup MongoDB Replica Set and PBM for each replica member on client node"
```
Comment (puneet0191) [inline pmm-tests/pmm-framework.sh:109]: Please correct this format, generally cli params use - and not _

File: pmm-tests/pmm-framework.sh [hunk 2]
```diff
@@ -2633,6 +2638,20 @@ setup_remote_db_docker_compose () {
...
 }
 
+setup_mongo_replica_for_backup() {
+  echo "Setting up MongoDB replica set with PBM"
+  sudo percona-release enable pbm release && sudo yum -y install percona-backup-mongodb
+  setup_docker_compose
+  mkdir -p /tmp/mongodb_backup_replica || :
+  pushd /tmp/mongodb_backup_replica
+  if [ ! -d "pmm-ui-tests" ]; then
+    git clone --branch PMM-8715-mongodb-rs-setup https://github.com/percona/pmm-ui-tests
```
Comment (puneet0191) [inline pmm-tests/pmm-framework.sh:2648]: Fix branch name here.

**Review by puneet0191 — APPROVED** [review]
Fix both comments, looks good. Please remember to update aws parameter confluence. 

Great work. thank you

File: pmm-tests/pmm-framework.sh (hunk 1 above)
Comment (yurkovychv) [reply pmm-tests/pmm-framework.sh]: done thanks

File: pmm-tests/pmm-framework.sh (hunk 2 above)
Comment (yurkovychv) [reply pmm-tests/pmm-framework.sh]: done

**Pull request merged**